### ISSUE TITLE
Imdb "find" imdb scraprt check error

### ISSUE THREAD

**Issue by kolbdog323** [theMK2k/Media-Hoarder#73]
Here is what its giving me:

query '天気の子' [null] imageURL mismatch got: "https://m.media-amazon.com/images/M/MV5BYjEzMjJjNzctNmFmMy00MTE4LTkwZTItYjY3Y2VjZmYxNmE3XkEyXkFqcGdeQXVyOTI4MzgyNTk@._V1_.jpg" expected: "https://m.media-amazon.com/images/M/MV5BNzE4ZDEzOGUtYWFjNC00ODczLTljOGQtZGNjNzhjNjdjNjgzXkEyXkFqcGdeQXVyNzE5ODMwNzI@._V1_.jpg"

**Comment by kolbdog323** [comment]
looks like the sample image got updated on imdb website since i looked at the mvie page

**Comment by theMK2k** [comment]
correct, I also fixed that - I changed the check in a way that it looks if "at least an image is provided" and not "which exact URL the image has" - the check is a bit weaker that way but it will prevent warnings like this where actually everything is fine

next version will contain the fix

**Comment by kolbdog323** [comment]
Okay, thanks I  just wasnt sure if you were aware of that error or not because i tried adding a new movie and got that erorr.

**Comment by kolbdog323** [comment]
AN

> correct, I also fixed that - I changed the check in a way that it looks if "at least an image is provided" and not "which exact URL the image has" - the check is a bit weaker that way but it will prevent warnings like this where actually everything is fine
> 
> next version will contain the fix

And maybe if no image provided that is when the user can add its own poster?

**Comment by theMK2k** [comment]
> And maybe if no image provided that is when the user can add its own poster?

This is another topic :)
